frontend/src/components/opportunity/ZipDownloadButton.tsx

```diff
@@ -7,14 +7,15 @@ import { USWDSIcon } from "src/components/USWDSIcon";
 
 type ZipDownloadProps = {
   opportunityId: number;
+  testId?: string;
 };
 
 const downloadAttachmentsZip = (opportunityId: number) => {
```

Comment:
if you move this function out into a separate file it'd be easy to mock it out and test that it's being called correctly. That's the pattern we follow most often for this sort of thing, one bonus being you can also test the mocked function more easily.

otherwise, you could mock `window.open`, but that's slightly trickier. There is some prior art on that with the way we mock window methods in the sessionStorage tests